Services/CO.CDP.Login.WebApi/CO.CDP.Login.WebApi.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Please add this project to project to solution

![image](https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/assets/64859911/9e26f12d-d0fe-4479-8858-b57f182a5cd9)